erpnext/projects/doctype/time_log/time_log.py

```diff
@@ -58,16 +60,20 @@ def set_status(self):
 
 	def set_title(self):
 		"""Set default title for the Time Log"""
-		if self.title:
-			return
-
-		from frappe.utils import get_fullname
+		
 		if self.production_order:
 			self.title = _("{0} for {1}").format(self.operation, self.production_order)
-		elif self.activity_type and (self.task or self.project):
-			self.title = _("{0} for {1}").format(self.activity_type, self.task or self.project)
-		else:
-			self.title = self.task or self.project or get_fullname(frappe.session.user)
+		elif self.activity_type :
+ 			self.title = _("{0}").format(self.activity_type)
+ 		
+ 		if self.quotation_:
```

Comment:
we have some customised fields, when we pull to standard, we have to do another version to add these customised fields to json file.